modules/ROOT/pages/_partials/push/handling-push-notifications.adoc

```diff
@@ -54,14 +54,14 @@ Cordova::
 
 NOTE: You can only develop push applications using Ionic. Plain Cordova is not supported.
 
-. Install the plugin:
-+
 [source,bash]
 ----
-$ npm install --save @ionic-native/push
+pushObject.on('notification').subscribe(data => {
```

Comment:
could we reinstate the steps, something like:
. Add the following code to your app:
<code>
. Review the documentation...

Reply:
agreed